kolibri/content/api.py

```diff
@@ -0,0 +1,222 @@
+"""
+This module acts as the only interface point between other apps and the database backend for the content.
+It exposes several convenience functions for accessing content
+"""
+from kolibri.content import models as KolibriContent
```

Comment:
I can't answer to how Sublime works, but you might have to configure a save
action or find the keyboard shortcut for auto-sorting imports.

On 2 February 2016 at 03:06, Eli <notifications@github.com> wrote:

> In kolibri/content/api.py
> <https://github.com/learningequality/kolibri/pull/14#discussion_r51513069>
> :
>
> > @@ -0,0 +1,222 @@
> > +"""
> > +This module acts as the only interface point between other apps and the database backend for the content.
> > +It exposes several convenience functions for accessing content
> > +"""
> > +from kolibri.content import models as KolibriContent
>
> tried to install isort on my sublime 3, nothing changed
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/learningequality/kolibri/pull/14/files#r51513069>.
>